spec/unit/rds_spec.rb

```diff
@@ -367,6 +367,21 @@
     end
   end
 
+  describe 'when parameter_group_name provided' do
+    before(:context) do
+      @plan = plan(role: :root) do |vars|
+        vars.parameter_group_name = 'test-parameter-group'
+      end
+    end
+
+    it 'uses the provided value for the assigned parameter group' do
+      expect(@plan)
+        .to(include_resource_creation(type: 'aws_db_instance')
+              .with_attribute_value(:parameter_group_name,
+                                    'test-parameter-group'))
+    end
+  end
+
```

Comment:
Please add a "negative assertion" for the default case where a parameter_group_name is not provided. See for example this assertion:

```
    it 'does not set IOPS' do
      expect(@plan)
        .to(include_resource_creation(type: 'aws_db_instance')
              .with_attribute_value(:iops, a_nil_value))
    end
```